src/Prism.Core/Prism.Core.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net45;net47</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net45;net47;net5.0</TargetFrameworks>
```

Comment:
please revert the netstandard target to 2.0. There are too many consumers out there using netstandard2.0. We need to continue supporting netstandard2.0 for the foreseeable future.